CauseContract.sol

```diff
@@ -182,9 +187,22 @@ contract CauseContract {
         donate();
     }
 
-    // modifier to ensure only admin is able to call function
+    function getAdmin() public view returns (address) {
```

Comment:
There's no need to return just the admin if we can retrieve that directly from the contract instance `contract.admin`?

Reply:
Unfortunately the admin isn't accessible in this way, tested it and it can't compile the contract correctly without it being a return admin function.